docs/pages/careers.tsx

```diff
@@ -183,6 +177,12 @@ const openRolesData = [
           'You will strengthen the advanced components team, build new ambitious complex features, work on strategic problems, and help grow the adoption.',
         url: '/company/react-engineer/',
       },
+      {
+        title: 'React Engineer - X - Community support',
```

Comment:
To keep the title short and to keep the team convention for the end? 

```suggestion
        title: 'React Support Engineer - X',
```

x the other instances?

Reply:
We can add this change and see how it will go

> Now, I'm not sure that we can do much at the role level. We still have to call a cat a cat.

Agree, I don't want to mislead people but as far as I understand the person that will be in this position is still very much a React engineer it's just that they will focus primarily on the issues that people report. Like when you hear e-commerce you think Shopify for online goods I'm afraid that when the React developers we want to target see Support they think call center (or some other form of support that only involves rerouting issues, not coding ).